app/models/invoice.rb

```diff
@@ -8,6 +8,6 @@ class Invoice < ApplicationRecord
   enum status: ['in progress', 'completed', 'cancelled']
 
   def self.incomplete_invoices
-    Invoice.where(status: "in progress")
+    Invoice.where(status: "in progress").order(created_at: :asc)
```

Comment:
I was wondering if this would work better as a scope, because eventually I think we will need to sort by status then date within status. That might not apply to Invoices, but looking ahead at a merchant user story I think I saw that nested sort.